CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
     * Bump `paypal-messages` to version `1.0.3`
 * ShopperInsights (BETA)
     * Add `PageType.PAYMENT_METHOD_SELECTION` and `PageType.PAYMENT_METHOD_ADDITION`
+    * Add `updateCustomerSession` public method to `ShopperInsightsClientV2` (beta)
```

Comment:
```suggestion
    * Add `updateCustomerSession` public method to `ShopperInsightsClientV2`
```

Reply:
@tdchow looks like we missed adding an entry for `createCustomerSession`. Should we add that in this PR?

Reply:
> @tdchow looks like we missed adding an entry for createCustomerSession. Should we add that in this PR?

Might be good to add the missing entry in the PR. @sailaya99 - would you be able to add an entry for `createCustomerSession`?